doc/man7/EVP_KDF-HKDF.pod

```diff
@@ -17,11 +17,38 @@ of the KDF).
 
 The output is considered to be keying material.
 
+=head2 Fixed-Digest HKDF
+
+B<HKDF-SHA256>, B<HKDF-SHA384> and B<HKDF-SHA512> are fixed-digest versions
```

Comment:
You could potentially move this section into multiple places..

The first part and the reset comment could go after the list of these algorithm names
The part related to setting the digest could go into the "digest" item...